### PR TITLE
Docker addition

### PR DESCRIPTION
**Pull request by ysimillides**
Added docker image, and changed tests to use it.

### CODE REVIEW

**Comment by ysimillides** [comment]
This should also fix #12, #40 and #42

File: REQUIRE [hunk 1]
```diff
@@ -1,3 +1,3 @@
 julia 0.6
 PyCall
-Conda
+PyPlot
```
Comment (ChrisRackauckas) [inline REQUIRE]: Use Requires.jl?

Comment (ysimillides) [reply REQUIRE]: So it's only loaded when the plotting is called?

Comment (ChrisRackauckas) [reply REQUIRE]: So it's only loaded if the user uses PyPlot

**Comment by ChrisRackauckas** [comment]
Needs a README update to the install instructions

**Comment by ysimillides** [comment]
> Needs a README update to the install instructions

Will do. I've also noticed some floating print() statements in the tests when I was debugging which should be removed.

File: README.md [hunk 2]
```diff
@@ -17,11 +17,20 @@ DifferentialEquations.jl ecosystem. [Paraview](https://www.paraview.org/) can al
...
+2. Alternatively, one can install [Docker](https://www.docker.com/) and then run the following command  
+
+```sh
+docker run -ti ysimillides/fenics-julia-docker 
+```
+and once inside, 'julia' can be accessed by calling
+```sh
+julia
+```
+Once inside the julia environment, simply add FEniCS with Pkg.add("FEniCS"). All other dependencies are handled by the docker image.
```
Comment (ChrisRackauckas) [inline README.md]: Why not have it in the docker image?

**Comment by ChrisRackauckas** [comment]
I'm not sure if Requires works with include like that so I'd double-check it. I am also not sure if that's sufficient for the progress meter. But for the progress meter see http://people.physik.hu-berlin.de/~pfitzseb/blogposts/juno-on-07/ . Your next PR should be a v0.7 upgrade anyways and updating to the Base logging would be a good idea with that.

**Comment by ysimillides** [comment]
It works with include (I've tested it locally, and it's also in the slightly sparse documentation of Requires.jl). I'll try making a PR to Requires.jl to make it more explicit.  I'll take a look at the base logging.

**Comment by ChrisRackauckas** [comment]
Okay good! Let's move to v0.7 now!

**Comment by ysimillides** [comment]
Is PyCall working on v0.7 yet? It didn't last time for me, and the repo seems to have tests failing still.

**Pull request merged**